packages/EasyDecision/src/Bridge/Symfony/DataCollector/DecisionContextDataCollector.php

```diff
@@ -0,0 +1,80 @@
+<?php
+declare(strict_types=1);
+
+namespace EonX\EasyDecision\Bridge\Symfony\DataCollector;
+
+use EonX\EasyDecision\Interfaces\DecisionContextInterface;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\DataCollector\DataCollector;
+
+final class DecisionContextDataCollector extends DataCollector
+{
+    /**
+     * @const string
```

Comment:
I'm not sure that `@const` exist, let's use `@var`.

Reply:
Very noted @natepage ! 👍